add-on/chrome/test/xpcshell/test_loopapi_app_version.js

```diff
@@ -0,0 +1,19 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+"use strict";
+
+const { LoopAPI } = Cu.import("chrome://loop/content/modules/MozLoopAPI.jsm", {});
+var [, gHandlers] = LoopAPI.inspect();
+
+add_task(function test_mozLoop_npsSurveyURL() {
+  MozLoopService.initialize("1.3.2alpha");
+  let expectedVersion = MozLoopService.addonVersion;
+  // Test "alpha" has been stripped from string
+  gHandlers.GetAddonVersion({ data: [] }, result => {
+    Assert.equal(result, "1.3.2", "expected output should have stripped alpha from string");
+  });
+  // Test GetAddonVersion
+  gHandlers.GetAddonVersion({ data: [] }, result => {
+    Assert.equal(result, expectedVersion, "mozLoopService version expected to equal mozLoopAPI version");
```

Comment:
This test actually equates to a duplicate of the first. Maybe just drop it? Since the first test the code paths we're really interested in?